kubetest/client.py

```diff
@@ -215,6 +216,49 @@ def load_service(self, path, set_namespace=True):
             service.namespace = self.namespace
         return service
 
+    def load_persistentvolume(self, path, set_namespace=False):
+        """Load a manifest YAML into a PersistentVolume object.
+
+        By default, this will augment the PersistentVolume object with
+        the generated test case namespace. This behavior can be
+        disabled with the ``set_namespace`` flag.
```

Comment:
This comment does not match up with the default value of the `set_namespace` parameter. This comment should either be changed, or `set_namespace` should default to `True`.